### PR TITLE
where is this data from links

### PR DESCRIPTION
**Pull request by BibianaC**
relates to #642

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Compile theme
 In order to compile the sass theme run:
 
 ```
-sassc grantnav/frontend/sass/main.scss grantnav/frontend/static/css/theme.css
+pysassc grantnav/frontend/sass/main.scss grantnav/frontend/static/css/theme.css
```
Comment (Lathrisk) [inline README.md:111]: Good spot, I forgot to update this!

**Review by Lathrisk — APPROVED** [review]

**Comment by michaelwood** [comment]
One moment, this doesn't look correct on the dev site.

**Review by Lathrisk — APPROVED** [review]

**Comment by Lathrisk** [comment]
Approved, after testing on dev - http://grantnav-dev.default.threesixtygiving.uk0.bigv.io/grant/360G-tnlcomfund-0031040623

File: grantnav/frontend/templates/funder.html [hunk 2]
```diff
@@ -47,7 +47,7 @@ <h1 class="funder-header text-center">{{ funder.name }}</h1>
       </div>
     </div>
     <div class='spacer-4'>
-      <div class="box box--teal">
```
Comment (michaelwood) [inline grantnav/frontend/templates/funder.html:50]: I don't see a request in the issue to change the colour to black instead of teal?

**Review by michaelwood — CHANGES_REQUESTED** [review]
After recompiling the theme can you append to the link the date of recompiling in the base template.

```diff
diff --git a/grantnav/frontend/templates/base.html b/grantnav/frontend/templates/base.html
index 1caf49c..0878bae 100644
--- a/grantnav/frontend/templates/base.html
+++ b/grantnav/frontend/templates/base.html
@@ -19,7 +19,7 @@
         <link href="https://fonts.googleapis.com/css?family=Assistant" rel="stylesheet">
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 
-        <link rel="stylesheet" href="{% static 'css/theme.css'%}">
+        <link rel="stylesheet" href="{% static 'css/theme.css'%}?v=20210401">
         <link rel="stylesheet" href="{% static 'css/glyphicons-halflings.css'%}">
 
         <noscript>
```

This makes sure that the CSS is reloaded by users. I'd like to automate this and add a test for it in the future but for now we can do it manually when we update the compiled css. #666 and planio 28999.

File: grantnav/frontend/templates/funder.html (hunk 2 above)
Comment (BibianaC) [reply grantnav/frontend/templates/funder.html:50]: @michaelwood it is in the change that @drkane has done in the design system.

Comment (drkane) [reply grantnav/frontend/templates/funder.html:50]: Sorry - that was me changing how the templates are displayed in the design system. The box should stay teal.

Comment (BibianaC) [reply grantnav/frontend/templates/funder.html]: Thanks @drkane. I have now deleted the commit with these changes. I have checked locally and it is teal and underlined.

**Review by michaelwood — APPROVED** [review]

**Pull request merged**